roles/ceph-defaults/defaults/main.yml

```diff
@@ -503,8 +503,7 @@ mon_containerized_deployment: False # backward compatibility with stable-2.2, wi
 osd_containerized_deployment: False # backward compatibility with stable-2.2, will disappear in stable 3.1
 mds_containerized_deployment: False # backward compatibility with stable-2.2, will disappear in stable 3.1
 rgw_containerized_deployment: False # backward compatibility with stable-2.2, will disappear in stable 3.1
-containerized_deployment: "{{ True if mon_containerized_deployment or osd_containerized_deployment or mds_containerized_deployment or rgw_containerized_deployment else False }}" # backward compatibility with stable-2.2, will disappear in stable 3.1
-
+containerized_deployment: False
```

Comment:
Perhaps it should be part of a patch removing all the backward compatibility with stable-2.2 ?

Reply:
Without at least this much done in this PR the config won't validate, because ``containerized_deployment`` should be a boolean and when the validate module gets the config it's not expanded, so it reads as a string and fails validation.

I propose we remove the rest of the backwards compatibility stuff in a separate PR.

Reply:
Ok, I guess because of what you said above
> I did try to remove these and it caused a bunch of breakage in the roles. I think fixing that should be done in a separate PR.